core-services/prow/02_config/_config.yaml

```diff
@@ -378,6 +378,8 @@ tide:
     '*': https://prow.ci.openshift.org/pr
     openshift-priv: https://deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/pr
   priority:
+  - labels:
```

Comment:
I don't think we want to change this for everyone, and by editing the main file we do.

Reply:
I'm not sure we can change it only for our team

Reply:
This is tag that is added manually and can help the developers override the priorities of the repository pull-requests